README.md

```diff
@@ -42,37 +36,91 @@ func WriteReader(path string, input io.ReadSeeker) error
 ```
 WriteReader Calls DefaultClient's WriteReader method.
 
-#### type Client
+#### type AWSClient
```

Comment:
That's the conclusion I came to too. `Client` seems to make the most sense for the interface. I don't think AWSClient will be instantiated often given the existence of DefaultClient so I was less concerned about it.